docs_espressif/en/additionalfeatures/docker-container.rst

```diff
@@ -1,301 +1,305 @@
-    */
-    "mounts": [
-      "source=extensionCache,target=/root/.vscode-server/extensions,type=volume"
-    ],
-    /* follow the commands of Dockerfile to create the container
-    */
-    "build": {
-      "dockerfile": "Dockerfile"
-    },
-    /* Machine specific settings that should be copied into the container
-    */
-    "settings": {
-      "terminal.integrated.defaultProfile.linux": "bash",
-      "idf.espIdfPath": "/opt/esp/idf",
-      "idf.toolsPath": "/opt/esp",
-      "idf.gitPath": "/usr/bin/git"
-    },
-    /* An array of extensions that should be installed into the container. */
-    "extensions": ["espressif.esp-idf-extension"],
-    /* start the container with privileged mode, else the devices cannot be accessed on the docker image.
-    */
-    "runArgs": ["--privileged"]
-  }
-
-At this point, all related configurations have been completed.
+
+    {
+        /* A name for the dev container displayed in the UI */
+        "name": "ESP-IDF",
+        /* container name when creating container */
+        "image": "espressif/idf:latest",
+        /* mount the local folder to /workspaces folder of Docker image */
+        "workspaceMount": "source=${localWorkspaceFolder},target=/workspaces/project-name,type=bind",
+        /* the path of workspace folder, which means that this folder will be opened after container is running
+        */
+        "workspaceFolder": "/workspaces/project-name",
+        /* mount the VS Code extensions to the target path, and then they don't need to be installed again when rebuilding the container
+        */
+        "mounts": [
+            "source=extensionCache,target=/root/.vscode-server/extensions,type=volume"
+        ],
+        /* follow the commands of Dockerfile to create the container
+        */
+        "build": {
+            "dockerfile": "Dockerfile"
+        },
+        /* Machine specific settings that should be copied into the container
+        */
+        "settings": {
+            "terminal.integrated.defaultProfile.linux": "bash",
+            "idf.espIdfPath": "/opt/esp/idf",
+            "idf.toolsPath": "/opt/esp",
+            "idf.gitPath": "/usr/bin/git"
+        },
+        /* An array of extensions that should be installed into the container. */
+        "extensions": ["espressif.esp-idf-extension"],
+        /* start the container with privileged mode, otherwise the devices cannot be accessed on the Docker image.
+        */
+        "runArgs": ["--privileged"]
+    }
+
+All related configurations are now complete.
 
 Create a Container
-~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~
 
-1. Create a container and then start your development by clicking the ``><`` green button at the bottom left of Visual Studio Code and select **Open Folder in Container** to start creating a container.
+1.  To create a container and begin development, click the green ``><`` button at the bottom left of VS Code. Select **Open Folder in Container** to initiate the container creation.
```

Comment:
```suggestion
1.  To create a container and begin development, click the ``><`` button at the bottom left of VS Code. Select **Open Folder in Container** to initiate the container creation.
```
How about removing the button color here? The extension I'm using (v1.9.0) has a blue button.

![WeChatWorkScreenshot_12fc96ca-aca6-4c5b-9627-ef3cce2ef0e1](https://github.com/user-attachments/assets/0ca5adcc-e5f1-4607-b326-6ce64259febc)